light-client/verifier/src/config.rs

```diff
@@ -5,27 +5,81 @@
 use crate::{LightClientVerifier, TrustedValidatorSet};
 use mc_blockchain_types::{BlockID, BlockIndex};
 use mc_common::{NodeID, ResponderId};
-use mc_consensus_scp_types::{QuorumSet, QuorumSetMember, QuorumSetMemberWrapper};
 use mc_crypto_digestible::Digestible;
 use mc_crypto_keys::Ed25519Public;
 use prost::Message;
 use serde::{Deserialize, Serialize};
 use serde_with::{hex::Hex, serde_as};
 use std::{collections::BTreeSet, fmt, ops::Range};
 
+/// A version of `[QuorumSetMember]` that does not use an internally-tagged
+/// enum. The only reseaon this is needed is because we want to be able to use
+/// this configuration inside a cosmos wasm contract, and unfortunately the
+/// cosmwasm runtime does not suppoprt floating point operations. It turns out
```

Comment:
:upside_down_face: nit
```suggestion
/// enum. The only reason this is needed is because we want to be able to use
/// this configuration inside a cosmos wasm contract, and unfortunately the
/// cosmwasm runtime does not support floating point operations. It turns out
```